speckle/base/client.py

```diff
@@ -161,8 +138,8 @@ def login_with_token(self, token):
             'Authorization': token,
         })
 
-        if self.me is None:
-            self.me = {}
+        full_profile = self.accounts.get_profile()
+        self.me = {k: full_profile.get(k, None) for k in ('id', 'email', 'name', 'surname', 'company', 'avatar', 'role', 'apitoken')}
```

Comment:
We could maybe move this to the `SpeckleApiClient` constructor (`__init__(self, ...)`) so we're sure that there is always a `me` dictionary. Alternatively, using the schema objects, maybe we can define the response from `login()` as a schema object that the client stores.

Reply:
oh, the only problem is that in the `SpeckleApiClient` construction there is no user logged in yet.

Reply:
True, but we could still initialize an empty dict with all the right keys, so that calling something in `me` doesn't fail.